index.html

```diff
@@ -2514,7 +2496,7 @@ <h3>The <dfn>NFCReaderOptions</dfn> dictionary</h3>
           </li>
           <li>
             Set <var>ndef.PAYLOAD</var> to the <a>URL path</a> of the current
-            <a>browsing context</a>, <a href="#dfn-encode">encoded</a> as <a>UTF-16 encoding</a>.
+            <a>browsing context</a>.
```

Comment:
There's no such thing as current browsing context. A browsing context also has no URL path.

Reply:
We only allow this in top-level browsing context.

How do you suggest that I rewrite this?

Reply:
So you need to figure out what object you want the URL from and find a way from where you are now to there. I don't really see how browsing contexts relate here since they don't carry URLs.

Reply:
OK I will file an issue for this so we can fix it afterwards - this is a couple of other places

Reply:
Filed: https://github.com/w3c/web-nfc/issues/200